SDK Showcase/SDK Showcase/Interactors/CategoriesInteractor.swift

```diff
@@ -50,6 +50,12 @@ struct CategoriesInteractorReal: CategoriesInteractor {
                              Selection(name: "Cancel registration", disabled: true)],
                  requiredActions: [],
                  optionalActions: []),
+        Category(name: "User authentication",
+                 description: "Here you can see all registered authenticators for the registered user.",
+                 options: [],
+                 selection: interactor.userAuthenticatorOptionNames.map { Selection(name: $0) },
```

Comment:
pls prepare a simple method in the interactor for that (e.g. interactor.getAllRegisteredAuthSth)

Reply:
that is the method (perhaps should be renamed): `userAuthenticatorOptionNames`. Making names a Selection(s) is the job for `CategoriesInteracor`, isn't it?

Reply:
yes

Reply:
Done.

Reply:
I can change the name but here is mapping to `Selection` - that is the whole point to keep UI (`Selection`) here, not in the interactor.